readme.md

```diff
@@ -0,0 +1,30 @@
+![Polylang](assets/polylang-logo.png)
```

Comment:
Could we use only the Polylang logo (without Polylang in the image), write "Polylang" inside a `h1` tag and center the logo and the title?
May I also suggest to move teh image into a `.github` directory as `assets` has a special meaning for WP plugins ? See: https://plugins.trac.wordpress.org/browser/polylang/

Reply:
I will see what we can do for styling in markdown.

Reply:
There is no possibility to use CSS in Markdown for Github because it sanitize the markdown See https://github.com/github/markup
So I replace the PNG by a SVG image because SVG use a real text in its `<text>` tag See https://developer.mozilla.org/en-US/docs/Web/SVG/Element/text

I set a white background for github dark theme